app/layout/meta-tags/component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+export interface MetaTagsProps {
+  name: string,
+  title: string,
+  description: string,
+  url?: string,
+  type: string,
+  publisher?: string,
+  section?: string,
+  tag?: string,
+  image?: string,
+  imageURL?: string,
+  imageWidth?: string,
+  imageHeight?: string,
+  twitterCard?: string,
+  twitterImage?: string,
+  twitterSite?: string,
+}
+
+export const MetaTags: React.FC<MetaTagsProps> = ({
+  name,
+  title,
+  description,
+  url,
+  type,
+  publisher,
+  section,
+  tag,
+  image,
+  imageURL,
+  imageWidth,
+  imageHeight,
+  twitterCard,
+  twitterImage,
+  twitterSite,
+}: MetaTagsProps) => {
+  return (
+    <>
```

Comment:
You need to wrap these tags in a Head component from `import Head from 'next/head';`